meta/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 dependencies:
-  - role: elliotweiser.osx-command-line-tools
+  - src: 'elliotweiser.osx-command-line-tools'
```

Comment:
Actually, can we just use the even simpler format:

```yaml
dependencies:
  - elliotweiser.osx-command-line-tools
```

This format is the plaintext-style version, and unless you need to expand on more metadata (we don't in this case), it's more readable and easier to remember imo.